benchmarks/base/Dockerfile

```diff
@@ -1,6 +1,27 @@
-ARG PYTHON_VERSION=3.9-slim-buster
+FROM debian:buster-slim as base
 
-FROM python:${PYTHON_VERSION} as python
+ARG PYTHON_VERSION=3.9.6
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev \
+    git ca-certificates \
+    # cleaning up unused files
+    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+    && rm -rf /var/lib/apt/lists/*
+ENV PYENV_ROOT "/pyenv"
+ENV PATH "$PYENV_ROOT/bin:$PATH"
+RUN git clone https://github.com/pyenv/pyenv.git "$PYENV_ROOT"
+RUN pyenv install "$PYTHON_VERSION"
+
+FROM debian:buster-slim
```

Comment:
any reason to have as multi-build? doesn't seem like there is much to strip away, a few `lib*-dev` dependencies, but shouldn't make a big deal with the image?

Reply:
The difference is bigger than I would have expected: 821mb without multistage, 580mb with multistage.

Reply:
/shrug alright, interesting!